### DISCUSSION TITLE
Exclude new contributors with PRs that are excluded

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#8202] [Pull Requests] [answered]
The "New contributors" section is nice and all, but it doesn't make any real sense when it lists those who made PRs that are excluded from the changelog, because of the excluded labels configuration.

I have a configuration where we exclude PRs targeting a documentation branch because it becomes quite spammy.
And while the PRs themself are excluded from the changelog without issues do the new contributors still appear with those exact PRs that we excluded.

I understand the purpose of this section which is to thank new contributors and encourage others to participate, but given the current logic of excluding PRs should new contributors follow the same logic here. Otherwise are we required to manually clean up dozens of entries in the list and manually check each individual one for if their contribution was towards the docs or not.

**Comment by MylesBorins** [comment, marked as answer]
@Andre601 I an understand why it would make sense to remove a new contributor from the list if their PR is excluded from the release, but I think it is important for us to recognize those who have contributed. At this time I think the appropriate action is to leave things as they are. If we have more people reach out wanting to change this functionality we can revisit the decision

**Reply by Andre601** [reply]
Okay.

Could it be considered tho to like put the new contributors into a different category, depending on what their first PR was?
Like similar to putting the PRs into separate categories, have the contributors split up into separate categories to thank them individually for their main contributions.

Because we right now have it, where we thank people that contributed directly through code PRs and people that contributed to the documentation separately, which adds a lot more work as we now have to check who's PR was code and who's was documentation. So having a way to already sort this in some way would be cool.

**Reply by MylesBorins** [reply]
Unfortunately this is not something that we would be looking to explore, it seems like it would significantly complicate the overall experience and the flow you are describing is not one I have heard from other developers / customers
